cea/interfaces/dashboard/plots/static/js/dashboard-plots.js

```diff
@@ -109,7 +109,8 @@ function open_table(element) {
     let height = 500;
     let width = 900;
     let table = window.open(`../table/${element.dataset.dashboardIndex}/${element.dataset.plotIndex}`,
-        'popUpWindow', `height=${height},width=${width},location=no,menubar=no,status=no,titlebar=no,resizable`);
+        `table-${element.dataset.dashboardIndex}-${element.dataset.plotIndex}`,
+        `height=${height},width=${width},location=no,menubar=no,status=no,titlebar=no,resizable`);
     table.onload = function() {
```

Comment:
Is there a reason we're not doing this in a modal dialog? I think the UI would be better, no?

Reply:
the modal kinda blocks the plot so it would be difficult if the user wants to refer to it.